src/components/DownloadTableSimple/DownloadTableSimple.js

```diff
@@ -1,11 +1,51 @@
 import React from 'react';
+import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import { rem } from 'polished';
-import Table, { TableWrapper } from 'components/Table';
 import Link from 'components/Link';
+import Text from 'components/Text';
+import media from 'utils/media';
 
 import { COLORS, SPACE, BREAKPOINTS } from 'config';
 
+const Table = styled.table`
+  width: 100%;
+  min-width: ${rem(800)};
+
+  tr {
+    padding-bottom: ${rem(10)};
+  }
+
+  td, th {
+    height: ${rem(15)};
+
+    ${media.sm.css`
+      height: ${rem(20)};
+    `}
+  }
+
+  a {
+    color: ${COLORS.base};
+    text-decoration: none;
+  }
+
+  margin-bottom: 100px;
+`;
+
+const TableWrapper = styled(Text).attrs({
```

Comment:
is it possible to create and reuse one component, since code is the same?

Reply:
It might be able in future, after new design will be applied on the whole site. Currently, there are two versions of TableWrapper that are used in different places.

Reply:
Fixed